.gitignore

```diff
@@ -0,0 +1,4 @@
+bower_components
```

Comment:
Make sure to update this file to be the same as [metal-modal's](https://github.com/metal/metal-modal/blob/master/.gitignore).